pkg/cmd/cli/describe/describer.go

```diff
@@ -1369,7 +1369,8 @@ func describeBuildTriggerCauses(causes []buildapi.BuildTriggerCause, out *tabwri
 func squashGitInfo(sourceRevision *buildapi.SourceRevision, out *tabwriter.Writer) {
 	if sourceRevision != nil && sourceRevision.Git != nil {
 		rev := sourceRevision.Git
-		formatString(out, "Commit", fmt.Sprintf("%s (%s)", rev.Commit[:7], rev.Message))
+		commit := fmt.Sprintf("%s", rev.Commit[:(len(rev.Commit)/3)])
```

Comment:
@mfojtik yes, the build fails. oc describe will still panic, because it describes the failed build regardless. well, i did the if else etc, PTAL?